silo-vaults/contracts/MetaSilo.sol

```diff
@@ -16,24 +16,25 @@ import {
     PendingUint192,
     PendingAddress,
     MarketAllocation,
-    IMetaMorphoBase,
-    IMetaMorphoStaticTyping
-} from "./interfaces/IMetaMorpho.sol";
+    IMetaSiloBase,
+    IMetaSiloStaticTyping
+} from "./interfaces/IMetaSilo.sol";
 
 import {INotificationReceiver} from "./interfaces/INotificationReceiver.sol";
 import {IVaultIncentivesModule} from "./interfaces/IVaultIncentivesModule.sol";
 import {IIncentivesClaimingLogic} from "./interfaces/IIncentivesClaimingLogic.sol";
 
+
 import {PendingUint192, PendingAddress, PendingLib} from "./libraries/PendingLib.sol";
 import {ConstantsLib} from "./libraries/ConstantsLib.sol";
 import {ErrorsLib} from "./libraries/ErrorsLib.sol";
 import {EventsLib} from "./libraries/EventsLib.sol";
 
-/// @title MetaMorpho
+/// @title MetaSilo
 /// @author Forked with gratitude from Morpho Labs.
 /// @custom:contact security@morpho.org
```

Comment:
let's use silo email

Reply:
I opened a separate PR for this change https://github.com/silo-finance/silo-contracts-v2/pull/888